### PR TITLE
Fix crash when first time user opens .bsk with scripts.

### PR DESCRIPTION
**Pull request by diimdeep**
More info https://github.com/BespokeSynth/BespokeSynth/issues/133#issuecomment-932276570

### CODE REVIEW

**Comment by baconpaul** [comment]
So @awwbees @diimdeep and i were chatting about this on discord and comparing with #315 

I think this PR has the effect of "if you load a BSK file and have never loaded python it does the equivalent of clicking on start python". On a system with python this will mean you don't get a crash but on a system without python I think you would then try and init python and fail

#315 has the different effect of "if you load a BSK file and have never loaded python, the python doesn't show up and you get the click to start python window"

So basically if you blow away the python_installed and load a .bsk with ScriptModule in it

1. This PR would make it the same as if you clicked 'start python' without you clicking and will show the python
2. #315 will show you the 'click to start python' message without crashing and if you click it will start

So I think (think) #315 might be safer for non-python systems loading bsk files.

**Comment by diimdeep** [comment]
@baconpaul Thanks, now I think this is not right change to do.
